app/views/site/about.html.erb

```diff
@@ -103,7 +103,7 @@ board_members = [
     <div class="card blank"></div>
     <% else %>
     <div class="card">
-      <%= image_tag "board/#{member[:picture]}", alt: member[:name], class: 'card-img-top' %>
+      <%= image_tag "board/#{member[:picture]}", skip_pipeline: true, alt: member[:name], class: 'card-img-top' %>
```

Comment:
CI started throwing deprecation warnings about this that I couldn't reproduce locally. 🤷‍♂️